napari/layers/points/points.py

```diff
@@ -672,6 +692,38 @@ def current_size(self, size: Union[None, float]) -> None:
             self.refresh()
             self.events.size()
 
+    @property
+    def fixed_size(self):
+        """bool: maintain point size fixed regardless of zoom"""
+        return self._fixed_size
```

Comment:
I'm not 100% this name will always be interpreted correctly - I might interpret `fixed_size=True` as fixing the size in  terms of number of data pixels i.e. current behaviour of `fixed_size=False`

An alternative approach could be to have a `size_mode` which can be set to `'world'` or `'canvas'`. Thoughts @brisvag @sofroniewn ?

Reply:
I agree, this name is terrible :P I don't love having to type in a string, it would be nicer to have a simple bool toggle.

Reply:
I think clarity is important here, especially if you imagine exposing this in the UI? It could be nice to have two buttons like we have for autoscale behaviour -> this makes it very obvious to users what the appropriate strings are

Reply:
We could make it `fixed_canvas_size` to be clear. I believe the default here is "off", i.e. markers change in size depending on zoom level (scaling is true). It seems like that will be the desired behavior for a molecular visualization too. Is there a reason you want it turned on (scaling is false)? Again I'm just a bit worried about too many choices here. What does @kevinyamauchi think?

Reply:
The reason is stuff like #429; in general, there are many applications where the size of a point has no meaning in data space, so I think this is something useful to have exposed. I definitely agree that the default should stay as is! And I like `fixed_canvas_size`, it's quite clear how `True` would behave, unlike with `scaling` or `fixed_size`.

Reply:
ah yeah, i forgot about #429. Ok i can live with `fixed_canvas_size`